### PR TITLE
new images and self service description in guides

### PR DESCRIPTION
**Pull request by sergio-ocon**
Added a description on how to use a service (admin user) and order it through the catalog.

### CODE REVIEW

File: site/docs/get-started/create-service-item.md [hunk 1]
```diff
@@ -74,5 +106,14 @@ Once you are logged out, enter ***https://\[your IP ADDRESS\]\]/self\_service***
 interface, that can be used to isolate you from the nuances of the
 underlying infrastructure.
 
+![screenshot0052](/assets/images/docs/screenshot_0052.png){:.img-responsive}
+
+Self service interface allows an easier management of services and a easier design for end users than the operational interface
```
Comment (hayesr) [inline site/docs/get-started/create-service-item.md]: "_The_ Self Service interface allows _for_ easier … and _an_ easier design …" 

or "… a more simple design for end users compared to the operational UI." might read a little better.

Also, isn't supposed to be The _Service UI_ now? (no "self")

File: site/docs/get-started/index.md [hunk 2]
```diff
@@ -9,16 +9,16 @@ virtual machines, public clouds and containers.
...
 * Enforce **compliance** across the environment.
-* Optimize the performance and utilization of you environment.
+* **Optimize** the performance and utilization of you environment.
 
 ## Following this Guide
 
 In order to follow this guide, you will need:
 
-* Somewhere to run the ManageIQ software.
+* Some place to run the ManageIQ software.
```
Comment (hayesr) [inline site/docs/get-started/index.md]: Should we be clear and say "a _system_ to run the ManageIQ software" and  "an infrastructure to manage"?

**Comment by Fryguy** [comment]
I'm concerned this will get out of sync with the official documentation in manageiq_docs.  Do we need this much detail in a "getting started" guide?

cc @chessbyte

**Comment by sergio-ocon** [comment]
hi @fryguy. We need to make users capable of seeing the value of MIQ, without entering into details.
It is just the bare minimum detail needed to create a service and execute it.

File: site/docs/get-started/create-service-item.md [hunk 3]
```diff
@@ -37,35 +46,58 @@ Press ***Configuration → Add a New Catalog*** and fill the fields. As there
...
+
 ### Ordering a service####
 
-We could press the ***Order*** button in the catalog, the dialog will
-appear, and after you will have your service running.
+Go to the Service catalog again. Now you can see your catalog item as part of the catalog. You can order it pressing the ***Order*** below the icon.
+
+![screenshot0050](/assets/images/docs/screenshot_0050.png){:.img-responsive}
+
+A service item can consist on different elements, not only a VM, and will be triggered through a series of automation workflows.
```
Comment (sergio-ocon) [inline site/docs/get-started/create-service-item.md]: Polishing!!!!

File: site/docs/get-started/create-service-item.md [hunk 4]
```diff
@@ -37,35 +46,58 @@ Press ***Configuration → Add a New Catalog*** and fill the fields. As there
...
-We could press the ***Order*** button in the catalog, the dialog will
-appear, and after you will have your service running.
+Go to the Service catalog again. Now you can see your catalog item as part of the catalog. You can order it pressing the ***Order*** below the icon.
+
+![screenshot0050](/assets/images/docs/screenshot_0050.png){:.img-responsive}
+
+A service item can consist on different elements, not only a VM, and will be triggered through a series of automation workflows. 
+The dialog we created will be used to gather information from the customer, hiding all the complexity of the details needed to actually deploy any of the components.
+
+Services has their own life cycle, as well as their components, so you can see (if your user profiles allows it), the resources associated to it in each moment
```
Comment (sergio-ocon) [inline site/docs/get-started/create-service-item.md]: Rewritten!

File: site/docs/get-started/create-service-item.md [hunk 5]
```diff
@@ -37,35 +46,58 @@ Press ***Configuration → Add a New Catalog*** and fill the fields. As there
...
+
+Ordering consists on different phases that are outside of the scope of this guide.
```
Comment (sergio-ocon) [inline site/docs/get-started/create-service-item.md]: Next time I'll write it in Spanish :D

File: site/docs/get-started/index.md (hunk 2 above)
Comment (sergio-ocon) [inline site/docs/get-started/index.md]: Updated

**Pull request merged**